packages/block-library/src/quote/index.js

```diff
@@ -33,6 +34,7 @@ const blockAttributes = {
 	},
 	align: {
 		type: 'string',
+		default: 'is-left',
```

Comment:
Hmm, the `default: 'is-left'` is saying that newly created Quote blocks should automatically have the `.is-left` class assigned, therefore left-aligned styling applied.

This was the case before, as newly created Quote blocks had `style="text-align: left;"` automatically applied. So, nothing changes really. Should an editor re-open a previously saved left-aligned quote block it would not suddenly lose its styling. Instead, `style="text-align: left;"` would be removed and `.is-left` would get substituted.

With that said, it could potentially have regressions on prior blocks that were saved as `style="text-align: right;"` or `style="text-align: center;"`, because it would replace _those_ with `.is-left` automatically. 

I could likely work up an instance using the [`deprecated` block API](https://wordpress.org/gutenberg/handbook/block-api/deprecated-blocks/) to allow for graceful transition.

Reply:
This is not a change I'd recommend we do this close to 5.0 release, and it also needs to take RTL languages into account.

Reply:
I agree that there should be no class by default. It seems like the css way to do things in terms of this class being a 'modifier', and modifiers only being applied for a variation of a default. 

I also think the classes could be more verbose, e.g. 'is-right-aligned', 'is-center-aligned'.